SportShop/MainWindow.xaml

```diff
@@ -0,0 +1,76 @@
+﻿<Window x:Class="SportShop.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:SportShop"
+        xmlns:local1="clr-namespace:SportShop.View"
+        xmlns:model ="clr-namespace:SportShop.Model"
+        xmlns:viewmodel ="clr-namespace:SportShop.ViewModel"
+        mc:Ignorable="d"
+        Title="Sport Shop" Height="600" Width="1000">
+
+    <Window.DataContext>
+        <viewmodel:ProductViewModel/>
+    </Window.DataContext>
+    <Window.Resources>
+        <DataTemplate x:Key="SportTemplate" DataType="{x:Type model:Product}">
```

Comment:
No need to specify key and DataType. If key is specified we can connect this DataTemplate explicitly using StaticResource markup extension.
If DataType is specified, WPF connects DataTemplate implicitly by type.